src/api/public/apidocs-new/paths/my_notifications.yaml

```
@@ -6,14 +6,17 @@ get:
   parameters:
     - in: path
       name: project
-      required: false
       schema:
         type: string
     - in: path
       name: page
-      required: false
       schema:
         type: int
+    - in: path
+      name: show_maximum
+      schema:
+        type: int
```

```suggestion
        type: string
    - in: path
      name: show_maximum
      schema:
        type: string
```
According to [the documentation](https://swagger.io/docs/specification/data-models/data-types/#numbers), OpenAPI has two numeric types, `number` and `integer`. I would suggest to use `string`, as the current code doesn't strictly require an integer.